simulators/trin-bridge/src/constants.rs

```diff
@@ -0,0 +1,14 @@
+// Execution Layer hard forks https://ethereum.org/en/history/
```

Comment:
We already have these constants over inside `portal-interop/src/constants.rs`, is there any way we can import those? If not, I'd recommend moving these to a single location where they can be imported wherever they're needed in the codebase rather than copying the constants multiple times

Reply:
Going to handle it in a follow up PR. Currently there is no good one place to put this.

So I am thinking of hosting a crate on ethereum/portal-spec-tests with test utils crate. A issue with this is then you have to update that repo to use new utils in this repo. So it would only be used for non-simulator-specific constants, and be updated less frequently to eventually deduplicate constants.

We have hivesim-rust, but I wouldn't be this constants there as these are not related.

And there isn't really a good place to declare a crate in this repo for common utils expessially with our goals of being re-included into ethereum/hive, breaking the projects structure would be a no no.

This change didn't really feel like a blocker to merge because of the reasons above more of a nit-pick which doesn't have the highest priority. Well yes it is nice to have no duplication of common constants between simulators, this PR isn't the place to change that.

Reply:
Sounds good, in the future it's worth mentioning how you're going to resolve a comment before hitting the `resolve` button, otherwise to the other person it looks like you just ignored it. Also, it gives a chance for the other person to have input into the planned resolution.